metricbeat/tests/system/metricbeat.py

```diff
@@ -139,3 +140,26 @@ def parameterized_with_supported_versions(base_class):
     variants = supported_versions(versions_path)
     decorator = parameterized_class(['COMPOSE_ENV'], variants)
     decorator(base_class)
+
+
+def tag(tag):
```

Comment:
This is then used with `@metricbeat.tag`?

Reply:
Yes, you need to import `metricbeat` for that, but I think we do it in all metricbeat tests.

Reply:
Well, you need to pass a tag as parameter, e.g:
```
@metricbeat.tag('oracle')
def test_something(self):
    ...
```